addons/hr_work_entry/models/resource.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
```

Comment:
Missing a second line. If you don't have a linter, I advise you to install one ;)